src/java/main/org/apache/jute/compiler/JType.java

```diff
@@ -27,7 +27,7 @@
 	private String mCName;
     private String mCppName;
     private String mCsharpName;
-    private String mJavaName;
+    protected String mJavaName;
```

Comment:
@rakeshadr Apologies, I must not have been clear. There are no changes here that are for the "sake" of performance. In fact, my concern is that my changes will negatively impact performance. 

After revisiting this change I think it makes more sense for us just to ignore the EI and EI2 issues. I reverted the changes to the jute compiler and added org.apache.zookeeper.server.quorum.QuorumAuthPacket to the findbugs exclusions file.